docs/fr-FR/newbs_building_firmware_configurator.md

```diff
@@ -0,0 +1,105 @@
+# Configurateur de QMK
+
+Le [Configurateur de QMK](https://config.qmk.fm) est une interface graphique en ligne permettant de générer des fichiers "hex" du firmware de QMK.
+
+?> **S'il vous plaît, suivez les étapes suivantes dans l'ordre.**
+
+Regardez le [Tutoriel vidéo](https://youtu.be/tx54jkRC9ZY)
+
+Le configurateur de QMK fonctionne mieux avec Chrome et Firefox.
+
+!> **Les fichiers d'autres outils, tels que KLE ou kbfirmware ne seront pas compatibles avec le configurateur QMK. Ne les chargez pas, ne les importez pas. Le configurateur QMK est un outil DIFFERENT.**
+
+## Sélectionner votre clavier
+
+Cliquez la boîte déroulante et sélectionnez le clavier pour lequel vous voulez créer une keymap.
+
+?> Si votre clavier a plusieurs verisons, faites attention à utiliser la version correcte.
+
+Je vais le répéter, parce que c'est important
+
+!> **FAITES ATTENTION A UTILISER LA BONNE VERSION !**
+
+Si votre clavier est annoncé comme fonctionnant grâce à QMK mais n'est pas dans la liste, il y a des chances que le développeur ne l'ait pas encore fait, ou que nous n'avons pas encore eu le temps de le merger. Ajoutez un problème (issue) sur [qmk_firmware](https://github.com/qmk/qmk_firmware/issues) demandant le support de votre clavier, s'il n'y a pas de [Pull Request](https://github.com/qmk/qmk_firmware/pulls?q=is%3Aopen+is%3Apr+label%3Akeyboard) ouvert pour lui. Il y a aussi des clavier alimentés par QMK qui sont sur le compte GitHub du fabriquant, il est bon de le vérifier aussi.
+
+## Sélectionner la disposition de votre clavier
+
+Choisissez la disposition (layout) qui représente le mieux la keymap que vous voulez créer. Certains clavier n'ont pas encore assez de dispositions ou des dispositions incorrectes. Ils seront supportés dans le future.
+
+## Nom de la Keymap
+
+Appelez cette keymap comme vous voulez.
+
+?> Si vous rencontrez des problèmes lors de la compilation, il peut être utile de changer ce nom, il peut déjà exister dans le repositoire du firmware QMK.
```

Comment:
`repositoire` ? totally weird

Reply:
Yep, changed everywhere else by "dépôt" , forgot that one, will fix and check if there are still others